exporters/module_loader.py

```diff
@@ -48,14 +48,22 @@ def load_stats_manager(self, options, metadata, **kwargs):
         from exporters.stats_managers.base_stats_manager import BaseStatsManager
         return self._load_module(options, metadata, BaseStatsManager, **kwargs)
 
+    def load_write_buffer(self, module_name, **kwargs):
+        from exporters.write_buffers.base_buffer import BaseBuffer
+        module_type = BaseBuffer
+        instance = self._instantiate_class(module_name, **kwargs)
+        if not isinstance(instance, module_type):
+            raise TypeError('Module must inherit from ' + str(module_type))
```

Comment:
do we really need that any custom write buffer be a subclass of BaseBuffer? We don't do this type check in other classes loaded through options.